azure-pipelines.yml

```diff
@@ -110,17 +110,13 @@ jobs:
     displayName: 'E2E Test - CDN (only runs on release builds)'
     # This test only runs after deployment from a release branch and the new CDN version has been deployed
     # This check will run on the PR to merge the release branch back into main
-    condition: and(
-      eq(variables['Build.Reason'], 'PullRequest'),
-      startsWith(variables['System.PullRequest.SourceBranch'], 'release/'),
-      eq(variables['System.PullRequest.TargetBranch'], 'main')
```

Comment:
Why are these conditions being removed?

Reply:
Removed for testing E2E test on release.

Reply:
So not intended for check-in?